src/utils/fetchClient.ts

```diff
@@ -33,7 +34,9 @@ function request<T>(
 
 export const client = {
   get: <T>(url: string) => request<T>(url),
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
   post: <T>(url: string, data: any) => request<T>(url, 'POST', data),
```

Comment:
Avoid disabling eslint rules without a good reason. If you need to disable a rule, provide a comment explaining why.